src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/Window.cs

```diff
@@ -140,43 +140,14 @@ public string ClassName
             get
             {
                 StringBuilder windowClassName = new StringBuilder(300);
-                InteropAndHelpers.GetClassName(Hwnd, windowClassName, windowClassName.MaxCapacity);
+                var numCharachtersWritten = NativeMethods.GetClassName(Hwnd, windowClassName, windowClassName.MaxCapacity);
 
-                return windowClassName.ToString();
-            }
-        }
-
-        /// <summary>
-        /// Gets represents the Window Icon for the specified window
-        /// </summary>
-        public ImageSource WindowIcon
-        {
-            get
-            {
-                lock (_processIdsToIconsCache)
+                if (numCharachtersWritten == 0)
                 {
-                    InteropAndHelpers.GetWindowThreadProcessId(Hwnd, out uint processId);
```

Comment:
(Clarification) Did we remove this because we're using a fixed icon for the ww plugin?

Reply:
Yep.  Again, I just elected to remove unused code instead of fixing issues that I couldn't (easily) test.